res/controllers/Reloop-Ready-scripts.js

```diff
@@ -0,0 +1,1141 @@
+// eslint-ignore-next-line no-var
+var ReloopReady = {};
+
+/// The controller only offers Low, High and Gain Knobs,
+/// settings this to true will remap them to control Low, Mid, High instead.
+ReloopReady.threeBandEQ = false;
+
+ReloopReady.backlightButtons = true;
+
+/// scratch parameter, change to your liking
+ReloopReady.scratchParams = {
+    alpha: 1/8,
+    beta: (1/8)/32,
+};
```

Comment:
I think `ReloopReady.scratchParams = { ... };` is dead code and can be removed without impact.

Reply:
Yeah, I noticed that when converting the mapping to ES6 classes. Unfortunately that work has stalled and I don't know when I'll be able to finish it. I'll try my best to do so asap. That's also the reason why I haven't pushed any fixes to your previous suggestions yet.